next/components/Molecules/BranchDetails/ContactsAndOpeningHours.tsx

```diff
@@ -12,7 +12,7 @@ type ContactsAndOpeningHoursProps = {
 const ContactsAndOpeningHours = ({ branch, branches }: ContactsAndOpeningHoursProps) => {
   const { t } = useTranslation('common')
 
-  if (!branches || branches.length === 0) {
+  if ((!branches || branches.length === 0) && !branches) {
```

Comment:
```
if (!branches?.length && !branch)
```
- `(!branches || branches.length === 0)` can be simplified to `!branches?.length`
  - the `?` can be used instead of asking first if `branches` is defined

- the second part should be `!branch` instead of `!branches`  :)